api/tests/mocha/services/connected-user-log.spec.js

```diff
@@ -0,0 +1,27 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const connectedUserLogService = require('../../../src/services/connected-user-log');
+const db = require('../../../src/db');
+
+describe('Connected Users Log service', () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+  describe('saveLog()', () => {
+    it('should save a log', () => {
+      sinon.stub(db.medicLogs, 'get').returns(Promise.reject({ status: 404 }));
```

Comment:
There's a slightly cleaner version of this...
```suggestion
      sinon.stub(db.medicLogs, 'get').rejects({ status: 404 });
```